README.md

```diff
@@ -1,9 +1,29 @@
-# Overview
-This repository contains a python package to support programmatic interactions with an RF preselector. 
-The package defines an abtract Preselector class that defines a simple interface for any preselector. 
-Currently, the WebRelayPreselector class in web_relay_preselector is the only implementation of the Preselector.
-The WebRelayPreselector supports a preselector featuring an [x310 WebRelay](https://www.controlbyweb.com/x310/).
-The package requires a [sigmf-ns-ntia metadata file](https://Github.com/NTIA/sigmf-ns-ntia) that describes the Sensor preselector and a config file to describe the x310 settings for the rf paths specified in the 
+# Introduction
+As shown in the example diagram below, a preselector is a device connected between an antenna and a signal analyzer designed to improve the performance of an RF sensor. 
+A preselector may include a variety of components including, but not limited to, filters, amplifiers, 
+calibration sources, and switches to control the components through which the RF signal flows.
```

Comment:
Suggested wording: A preselector is a device, connected between an antenna and a signal analyzer, designed to improve the RF performance and capability of a sensor. As illustrated in the diagram below, it may include a variety of...